policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/request/Policy.java

```diff
@@ -39,27 +39,26 @@
  * service but not needed by the rest of the palisade services. That includes
  * separating the rules that need to be applied at the resource level or the record level.
  *
- * @param <RULE_DATA_TYPE> The Java class that the rules expect the records of
+ * @param <V> The Java class that the rules expect the records of
```

Comment:
As per my previous comment about this type of change

Reply:
reverted change